app/schema.graphql

```diff
@@ -525,6 +527,11 @@ type SpanEdge {
   cursor: String!
 }
 
+type SpanIOValue {
+  mimeType: String!
```

Comment:
Making this an enum will make the frontend code significantly safer and protectable. I can write exhaustive checks so that I make sure I handle all cases.